inst/rmarkdown/templates/cholera_outbreak/skeleton/skeleton.Rmd

```diff
@@ -101,16 +110,32 @@ linelist_cleaned <- linelist_raw
 # overwrite variable names with defined clean names
 # colnames(linelist_cleaned) <- cleaned_colnames
 
-# OPTIONAL: you can also change specific var names using the *rename* function
-# linelist_cleaned <- rename(linelist_cleaned, sex = gender)
+# OPTIONAL: below is just an example in case you want to specifically change a few names)
+# you can also change specific var names using the *rename* function
+# linelist_cleaned <- rename(linelist_cleaned, 
+#                            sex = gender, 
+#                            age = age_years)
 
-# only keep variables defined data dictionary 
-# you could alternatively write out all the names you want to keep e.g. c("sex", "age", ...)
-linelist_cleaned <- select(linelist_raw, linelist_dict$data_element_shortname)
 
-# switch variable contents to show what you see on data entry (rather than coded values) 
-linelist_cleaned <- switch_vals(df = linelist_cleaned, disease = "Cholera")
+# OPTIONAL: if you only want to keep certain variables - you can select these by name or column number
+# view the names of your vars and their column number using: names(linelist_cleaned)
+# this example keeps the first three columns as well as age_years and sex variables
+# linelist_cleaned <- select(linelist_cleaned, c(1:3, "age_years", "sex")
 
+
+
+# OPTIONAL: ONLY RUN THESE LINES IF YOUR DATA MATCHES THE DICTIONARY EXACTLY(!) AND YOU DONT WANT ANY EXTRA VARS
```

Comment:
I don't understand why this block is here. It shouldn't matter if people have extra columns in their data frame as long as they have the variables used in this analysis.

Reply:
Was because annick had extra columns and then the renaming and switchvars went wampwamp and left columns empty, no good.

Reply:
I understand how swtich_vars panicked.... `match()` will return NA for non-matches, which gets messed up with subsetting. I've modified it to do 0 for non-matches so that they are replaced respectfully.